py-develop-plugins.md

```diff
@@ -2,21 +2,21 @@
 
 Python plugins use the [JSON-RPC](https://flow-launcher.github.io/docs/#/json-rpc) protocol to communicate with Flow via JSON structured calls.
 
-When building a Python plugins there are several things to be mindful of:
+When building a Python plugins, there are several things to be mindful of:
 
 * The most important thing is we do not expect users to have to manually install the dependencies in requirements.txt because we aim to provide a seamless experience for them. This can be achieved by adding the following three things to your project:
-    1. Add a GitHub workflow- use a GitHub workflow that will install all your plugin's dependencies including the Python flowlauncher module to a folder called Lib inside your plugin.
-    2. Publish all as a zip- zip up your project including a lib directory that contains the modules and publish it to GitHub Releases page.
-    3. Point your module imports to the lib directory- reference all the modules to that directory where they are first imported.
+    1. Add a GitHub workflow — use a GitHub workflow that will install all your plugin's dependencies including the Python flowlauncher module to a folder called Lib inside your plugin.
```

Comment:
would ":" better than "-"?

Reply:
I honestly don't know, both options look okay to me. Should I replace them with `:`?